File: SamaraOrganics-SPA/src/app/_models/CreateInvoiceVM.ts
import { AccountsList } from './AccountsList';
import { VendorsDto } from './VendorsDto';
import { CategoryDto } from './CategoryDto';
import { StatusDto } from './StatusDto';

export interface CreateInvoiceVM {
    VendorsList: VendorsDto[];
    CategoryList: CategoryDto[];
    StatusList: StatusDto[];
    AccountsList: AccountsList[];
}

File: SamaraOrganicsSystem/Models/ViewModels/CreateInvoiceVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using SamaraOrganicsSystem.Dtos;

namespace SamaraOrganicsSystem.Models.ViewModels
{
    public class CreateInvoiceVM
    {
        public IEnumerable<VendorDto> VendorList { get; set; }
        public IEnumerable<CategoryDto> CategoryList { get; set; }
        public IEnumerable<StatusDto> StatusList { get; set; }
        public IEnumerable<AccountsDto> AccountsList { get; set; }
       
    }
}
